CHANGELOG.md

```diff
@@ -0,0 +1,11 @@
+# Change Log
+All notable changes to this project will be documented in this file.
+
+##[2.7.0] - 2017-02-16
```

Comment:
https://github.com/3scale/3scale_ws_api_for_php/releases last release was 2.6.2. I thought next should be following the sequence - 2.6.3